platform/web-security/src/main/java/org/motechproject/security/constants/EventSubjects.java

```diff
@@ -0,0 +1,16 @@
+package org.motechproject.security.constants;
+
+/**
+ * Utility class used for storing event subjects used by the security module.
+ */
+public final class EventSubjects {
```

Comment:
why are these subjects separate from the previous ones? We should have them in one place. The comment about event subject naming also applies here.